Makefile

```diff
@@ -133,6 +133,13 @@ test: manifests generate fmt vet envtest ## Run tests.
 test-e2e:
 	go test ./test/e2e/ -v -ginkgo.v
 
+.PHONY: markdownlint
+markdownlint:
+	@$(CONTAINER_TOOL) run --platform linux/amd64 -v .\:/md\:z  docker.io/peterdavehello/markdownlint  markdownlint --dot --ignore ./vendor --dot --config .markdownlint.yaml ./
+
+.PHONY: markdownlint-fix
+markdownlint-fix:
+	@$(CONTAINER_TOOL) run --platform linux/amd64 -v .\:/md\:z  docker.io/peterdavehello/markdownlint  markdownlint --dot --ignore ./vendor --dot --fix --config .markdownlint.yaml ./
```

Comment:
can we use `https://github.com/DavidAnson/markdownlint`  https://hub.docker.com/r/davidanson/markdownlint-cli2 official docker image from this repo?

Reply:
done  but I had to adjust the invocation command line and the cli2 does not seem to understand the `--ignore`option so `make markdownlint` fails locally for me on some files under `vendor`...